Assets/Chunk.cs

```diff
@@ -15,12 +15,12 @@ public class Chunk
     Block[][][] Blocks { get; }
     private MeshBuffer buffer;
     public bool Dirty { get; private set; }
+    public bool Generating { get; set; }
```

Comment:
Wouldn't this be resolved by letting the update handle it queued now?

Reply:
Sadly no, since `SetBlock` still calls `SetDirty`, causing it to be enqueued every block update